interface.html

```diff
@@ -59,6 +59,37 @@
     </div>
   </div>
 
+  <h3></h3>
+  <div class="form-group clearfix">
+    <div class="col-sm-4 control-label">
+      <label for="loggedInUserTime">Keep the user logged in for</label>
+      <a class="help-icon" data-toggle="tooltip" data-placement="top" title="Force user re-authentication after a number of hours">
```

Comment:
This should use the same markup as the other checkbox, with the `<a>` inside the `<label>`

Reply:
Did the changes as per you suggestion